### DISCUSSION TITLE
no connection on AWS

### DISCUSSION THREAD

**Discussion by slln** [wg-easy/wg-easy#255] [General]
Installed wg-easy on an AWS instance, can access it but generating the new clients does not respect/ connect to the WG installation.
I've used PiVPN and WG is at 10.6.0.0/24 whilst wg-easy seems to create clients at 10.8.0.0/24 sequentially 
If I force wg-easy address to 10.6.0.1 then I'm "allowed" only one client and the tunnel is anyway not functional.

![screenshot 2022-05-06 at 21 55 41](https://user-images.githubusercontent.com/48659657/167200699-20f92e8b-7608-495b-a531-aae6273963d4.png)

Q: how do I "connect" wg-easy to the WG install?

LE:
If I edit the conf file generated by wg-easy with a manual 10.6.0.X address not the 10.Y.0.X address generated by wg-easy, all traffic except web is working.

Any Ideas?

**Comment by pheiduck** [comment]
@slln please try the newest image: `ghcr.io/wg-easy/wg-easy:latest`
